extra/utils/validation.js

```diff
@@ -0,0 +1,59 @@
+const users = [{
+    traineeEmail: 'trainee1@successive.tech',
+    reviewerEmail: 'reviewer1@successive.tech',
+    },
+    {
+        traineeEmail:'urvashi.chaudhary@successive.tech',
+        reviewerEmail:'mohammad.adil@successive.tech',
+    },
+    {
+        traineeEmail:'i@gmail.com',
+        reviewerEmail:'jfjjjjf',
+    },
+];
+
+
+function validateEmail(input){
+    let correct = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+    
+    return correct.test(input);
+}
+
+
+
+
+function validateUser(users){
+    let valid=0;
+    let invalid=0;
+    let v_users="";
+    let i_users="";
+    let len=users.length;
+    //console.log(len);
+    for(let i=0;i<len;i++){
```

Comment:
must use an array method to iterate over an array.

Reply:
done